Mathlib/Algebra/Group/Defs.lean

```diff
@@ -13,29 +13,29 @@ import Mathlib.Data.List.Basic
 # Typeclasses for (semi)groups and monoids
 
 In this file we define typeclasses for algebraic structures with one binary operation.
-The classes are named `(add_)?(comm_)?(semigroup|monoid|group)`, where `add_` means that
-the class uses additive notation and `comm_` means that the class assumes that the binary
+The classes are named `(Add)?(Comm)?(Semigroup|Monoid|Group)`, where `Add` means that
+the class uses additive notation and `Comm` means that the class assumes that the binary
 operation is commutative.
 
 The file does not contain any lemmas except for
 
 * axioms of typeclasses restated in the root namespace;
 * lemmas required for instances.
 
-For basic lemmas about these classes see `algebra.group.basic`.
+For basic lemmas about these classes see `Algebra.Group.Basic`.
 
-We also introduce notation classes `has_smul` and `has_vadd` for multiplicative and additive
+We also introduce notation classes `HasSmul` and `HasVadd` for multiplicative and additive
```

Comment:
These should be `SMul` and `VAdd`.

Reply:
Do you mean in the doc or in mathlib itself? At line 52 there is
```
/-- Typeclass for types with a scalar multiplication operation, denoted `•` (`\bu`) -/
@[ext, to_additive]
class HasSmul (M : Type _) (α : Type _) where
  smul : M → α → α
```

Reply:
I was thinking mathlib itself. In Lean 4 all the notation typeclasses have names like `HAdd`, `Mul`, etc. This PR doesn't need to be the place to change them, but it could be if you'd like to.

Reply:
Done. I've also modified `AddMonoid.HasSmul` to `AddMonoid.Smul` (and similarly for other declarations).